templates/openstack/what-is-openstack.html

```diff
@@ -343,207 +336,92 @@ <h2>Companies involved in OpenStack</h2>
       </div>
     </div>
   </div>
+  <div class="u-fixed-width">
+    <p>
+      Canonical is the founding member of the foundation, official maintainer
+      and publisher of OpenStack packages on Ubuntu, leader of the
+      <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms project</a>
+      and one of the biggest contributors to OpenStack all the time.
+    </p>
+  </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-8">
+<section class="p-strip">
```

Comment:
Maybe this section to `p-strip--light` and the other sections below the other way around? 

```suggestion
<section class="p-strip--light">
```